lib/container/HoistContainerReferencesPlugin.js

```diff
@@ -0,0 +1,205 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Zackary Jackson @ScriptedAlchemy
+*/
+
+"use strict";
```

Comment:
This plugin needs to be applied still somewhere in the compilation.
https://github.com/module-federation/core/blob/main/packages/enhanced/src/lib/container/runtime/FederationRuntimePlugin.ts like i do here.